dependency-check-suppressions.xml

```diff
@@ -1,224 +1,224 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It looks like the formatting of this file has changed!  That will make it difficult to identify small changes.